config/hsc/processCcd.py

```diff
@@ -19,6 +19,7 @@
 config.calibrate.astrometry.matcher.maxRotationDeg=1.145916
 config.calibrate.astrometry.wcsFitter.numRejIter=3
 
+config.charImage.astrometry.wcsFitter.order = 3
 config.calibrate.astrometry.wcsFitter.order = 3
```

Comment:
Or:

    for sub in (config.charImage, config.calibrate):
        sub.astrometry.wcsFitter.order = 3